delfin/tests/unit/drivers/hpe/hpe_msa/test_hpe_msastor.py

```diff
@@ -0,0 +1,101 @@
+import sys
+import paramiko
+
+from delfin import context
+from unittest import TestCase, mock
+from delfin.tests.unit.drivers.hpe.hpe_msa import test_constans
+from delfin.drivers.utils.ssh_client import SSHPool
+from delfin.drivers.hpe.hpe_msa.ssh_handler import SSHHandler
+from delfin.drivers.hpe.hpe_msa.hpe_msastor import HpeMsaStorDriver
+
+sys.modules['delfin.cryptor'] = mock.Mock()
+
+ACCESS_INFO = {
+    "storage_id": "kkk",
+    "ssh": {
+        "host": "110.143.132.231",
+        "port": 22,
+        "username": "user",
+        "password": "pass",
+        "pub_key": "ddddddddddddddddddddddddd"
+    }
+}
+
+
+class Request:
```

Comment:
Please confirm this class has been used.

Reply:
delete this class function